src/index.ts

```diff
@@ -1,3 +1,5 @@
-import { DynamoDB } from './lib/dynamodb';
+import { DynamoDB } from './lib';
 
-export = new DynamoDB();
+export { ReadItem, TransactRead, TransactQuery, TransactDeleteItem, TransactInsertItem, TransactUpdateItem, WriteItem, TransactWrite, TransactMethod, BaseQuery, CreateTable, DeleteItem, DeleteTable, Executable, InsertItem, ListTables, Method, Query, Scan, UpdateItem, Map, Params, QueryBuilder, ThroughPut, AttributeDefinition, KeySchema, Projection, GlobalSecondaryIndex, LocalSecondaryIndex, Schema, UpdateQuery, KeyNameResult, ValueNameResult, ParseResult, ParseResult, Options, DynamoDB, TableOptions, Table } from './lib';
```

Comment:
Can you spread this over multiple lines? I believe it improves readability.

Not sure about exporting everything though. I think the method's on their own would be a good start.

If it's not directly usable publicly, we shouldn't export it. I believe the `ParseResult` and the `ValueNameResult` are types which are only used internally if I'm not mistaken?